code/modules/mob/observer/freelook/chunk.dm

```diff
@@ -23,6 +23,7 @@
 	var/image/obfuscation = obfuscation_images[T]
 	if(!obfuscation)
 		obfuscation = image(icon, T, icon_state)
+		obfuscation.plane = EFFECTS_ABOVE_LIGHTING_PLANE
```

Comment:
Would you happen to have a screenshot of how the obfuscation looks now handy?

Reply:
The problem here is what obfuscation level was lower than lighting, causing meta-tracking by lighting above this effect.

Originally, this should look like this now:
![image](https://user-images.githubusercontent.com/44920739/75564071-7f73ad80-5a5c-11ea-889a-7659995c92ed.png)

Reply:
Ah looks good.